src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -441,38 +439,110 @@
+                         :gasFeeMode           gas-rates
+                         :fromLockedAmount     from-locked-amount}
+                        rest-keys)]]
      {:db            (cond-> db
                        :always (assoc-in [:wallet :ui :send :amount] amount)
                        :always (assoc-in [:wallet :ui :send :loading-suggested-routes?] true)
                        :always (assoc-in [:wallet :ui :send :sender-network-values]
                                 sender-network-values)
                        :always (assoc-in [:wallet :ui :send :receiver-network-values]
                                 receiver-network-values)
-                       :always (assoc-in [:wallet :ui :send :suggested-routes-call-timestamp] now)
                        :always (update-in [:wallet :ui :send] dissoc :network-links)
                        token   (assoc-in [:wallet :ui :send :token] token))
-      :json-rpc/call [{:method     "wallet_getSuggestedRoutes"
-                       :params     request-params
-                       :on-success (fn [suggested-routes]
-                                     (rf/dispatch [:wallet/suggested-routes-success suggested-routes
-                                                   now]))
-                       :on-error   (fn [error]
-                                     (rf/dispatch [:wallet/suggested-routes-error error])
-                                     (log/error "failed to get suggested routes"
-                                                {:event  :wallet/get-suggested-routes
-                                                 :error  error
-                                                 :params request-params}))}]})))
+      :json-rpc/call [{:method   "wallet_getSuggestedRoutesV2Async"
+                       :params   params
+                       :on-error (fn [error]
+                                   (rf/dispatch [:wallet/suggested-routes-error error])
+                                   (log/error "failed to get suggested routes (async)"
+                                              {:event  :wallet/get-suggested-routes
+                                               :error  error
+                                               :params params}))}]})))
+
+(rf/reg-event-fx :wallet/stop-get-suggested-routes
+ (fn []
+   {:json-rpc/call [{:method     "wallet_stopSuggestedRoutesV2AsyncCalcualtion"
+                     :params     []
+                     :on-success (fn []
+                                   (rf/dispatch [:wallet/clean-routes-calculation]))
+                     :on-error   (fn [error]
+                                   (log/error "failed to stop suggested routes calculation"
+                                              {:event :wallet/stop-get-suggested-routes
+                                               :error error}))}]}))
+
+(defn- transform-new-to-old-path
+  [new-path]
+  (let [unit            (-> (get-in new-path [:from-token :symbol])
```

Comment:
This function could be moved to a data store namespace and unit tested, at least one happy path test.